### DISCUSSION TITLE
Where are zeal doc files kept ?

### DISCUSSION THREAD

**Discussion by crillion** [zealdocs/zeal#1357] [Q&A] [answered]
Hi,

   I'd like to know where are zeal's files kept. I've installed it on windows 10 (zeal version 0.6.1),  but I can't see a folder with all its files.

   Thanks,
              Marco

**Comment by 12beesinatrenchcoat** [comment]
If you're referring to where the Docsets are, open Preferences (Edit > Preferences, or `Ctrl` + `,`, or the "Preferences" button on the welcome page), and a Docset storage directory should be listed there near the bottom.

**Comment by andre-st** [comment]
If you have already uninstalled Zeal, searched the web and ended up here:
Using the Flatpak package on Manjaro Linux, 
docsets were stored in my home directory:
`~/.var/app/org.zealdocs.Zeal/data/Zeal/Zeal/docsets`

**Comment by trollixx** [comment, marked as answer]
It's an old post but for anyone looking on Windows, the docsets are stored in `C:\Users\<USER>\AppData\Local\Zeal\Zeal\docsets` by default.
